textattack/models/wrappers/pytorch_model_wrapper.py

```diff
@@ -21,6 +21,7 @@ def __init__(self, model, tokenizer, batch_size=32):
     def __call__(self, text_input_list):
         model_device = next(self.model.parameters()).device
         ids = self.tokenize(text_input_list)
+        ids = [i["input_ids"] for i in ids]
```

Comment:
Please revert this line. `"input_ids"` is a key that belongs to the inputs of models that use Hugging Face tokenizers.  So I'm guessing that the model is being imported from the file using the PytorchModelWrapper when it's supposed to be a HuggingFaceModelWrapper. I think if you fixed that then this 'issue' as well as the one in `tensor.py` should fix itself.

Reply:
Gotcha, Reverted